modules/compiler/source/base/source/module.cpp

```diff
@@ -238,7 +238,8 @@ static bool loadKernelAPIHeader(clang::CompilerInstance &compiler,
   // stored inside the PCH file.
   llvm::BitstreamCursor &Cursor = moduleFile->InputFilesCursor;
   clang::SavedStreamPosition SavedPosition(Cursor);
-  if (Cursor.JumpToBit(moduleFile->InputFileOffsets[0])) {
+  uint64_t Base = multi_llvm::GetInputFilesOffsetBase(moduleFile);
```

Comment:
Honestly since it's so localized to this one function, I'd be tempted to do this without `multi_llvm` and instead have something like:

``` cpp
uint64_t Base = 0;
#if LLVM_VERSION_GREATER_EQUAL(18, 0)
Base = File->InputFilesOffsetBase;
#endif
```

We do this in other places, even in this same file.